mote/templates/event_summary.html

```diff
@@ -30,7 +30,7 @@
   </div>
 </div>
 
-<div class="evt-smry-body container p-3 pb-0">
+<div class="evt-smry-body container-fluid p-3 pb-0">
```

Comment:
Hi @darknao, could you please resolve the merge conflicts which were introduced as a result of #386 getting merged?

Reply:
done